src/components/EventRegistrantsModal/Modal/EventRegistrantsModal.tsx

```
@@ -128,82 +130,77 @@ export const EventRegistrantsModal = (props: ModalPropType): JSX.Element => {
+          </>
+        }
+      >
+        <Autocomplete
+          id="addRegistrant"
+          onChange={(_, newMember): void => {
+            setMember(newMember);
           }}
+          noOptionsText={
+            <div className="d-flex">
+              <p className="me-2">{t('noRegistrationsFound')}</p>
+              <span className={styles.onspotLink} onClick={() => setOpen(true)}>
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)